recipe/meta.yaml

```diff
@@ -10,20 +10,20 @@ source:
   md5: b858301ef92269c3e4de82d90cba3020
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - pyqt  # pyface requires pyqt, pyside or wxpython
+    - pyqt 4.11.*  # pyface requires pyqt, pyside or wxpython
     - traits
     - pygments
   run:
     - python
     - traits
-    - pyqt
+    - pyqt 4.11.*
```

Comment:
If this is needed, we might need to pull the old versions. This is needed as the `conda` solver prioritizes the latest overall set of versions over build numbers. As a consequence it will downgrade to old unpinned packages to get the latest PyQt and Qt versions. IOW this pinning doesn't appear to be respected currently.

FWIW the old packages will still be available, but under the `broken` label.

Any thoughts or questions about this?